### DISCUSSION TITLE
4.4.2 broken rendering of endash

### DISCUSSION THREAD

**Discussion by bmerry** [astropy/astropy-v5.0-paper#78] [Text comments] [answered]
The source contains `\textasciitilde $40--100\,\mathrm{ns}$` but the `--` doesn't render as an endash is math mode.

**Comment by pllim** [comment]
Feel free to PR!

**Comment by bmerry** [comment]
I don't have a conda environment, so I'm not able to actually build and check the rendering of a fix.

Note that this isn't just a case of the dash being slightly the wrong size. You get two dashes instead of one. It's easy to miss because one is at the end of a line and the next is at the start of the following line.

**Comment by adrn** [comment, marked as answer]
I fixed this in a commit that I will push soon - thanks @bmerry
